style.css

```diff
@@ -592,8 +348,185 @@ form textarea {
   border: 1px solid #5e6c84;
 }
 
-form img {
-  margin-bottom: 10px;
-  height: 3px;
-  border-radius: 8px;
+form ul li {
+  list-style: none;
+}
+
+#form-btn {
+  align-self: flex-start;
+  margin-left: 17%;
+
+  /* margin: 0 auto; */
+
+  /* display: inline-block; */
```

Comment:
- [ ] Please always make sure you clean up your codebase by removing the commented lines of code, this is to make sure your code is organized in a professional way and follows good standards.